src/patternfly/components/Menu/menu.scss

```diff
@@ -489,298 +417,217 @@
   }
 }
 
-// Header
-.#{$menu}__header {
-  --#{$menu}__item--PaddingTop: var(--#{$menu}__header--c-menu__item--PaddingTop);
-  --#{$menu}__item--PaddingRight: var(--#{$menu}__header--c-menu__item--PaddingRight);
-  --#{$menu}__item--PaddingBottom: var(--#{$menu}__header--c-menu__item--PaddingBottom);
-  --#{$menu}__item--PaddingLeft: var(--#{$menu}__header--c-menu__item--PaddingLeft);
-
-  padding-block-start: var(--#{$menu}__header--PaddingTop);
-  padding-block-end: var(--#{$menu}__header--PaddingBottom);
-  padding-inline-start: var(--#{$menu}__header--PaddingLeft);
-  padding-inline-end: var(--#{$menu}__header--PaddingRight);
-
-  > .#{$menu}__item {
-    --#{$menu}__item--BackgroundColor: var(--#{$menu}__header--c-menu__item--BackgroundColor);
-
-    margin-block-start: var(--#{$menu}__header--c-menu__item--MarginTop);
-    margin-block-end: var(--#{$menu}__header--c-menu__item--MarginBottom);
-    margin-inline-start: var(--#{$menu}__header--c-menu__item--MarginLeft);
-    margin-inline-end: var(--#{$menu}__header--c-menu__item--MarginRight);
-
-    &:hover {
-      --#{$menu}__item--BackgroundColor: var(--#{$menu}__header--c-menu__item--hover--BackgroundColor);
-    }
-
-    &:focus {
-      --#{$menu}__item--BackgroundColor: var(--#{$menu}__header--c-menu__item--focus--BackgroundColor);
-    }
-  }
-
-  + .#{$menu}__search {
-    --#{$menu}__search--PaddingTop: var(--#{$menu}__header__search--PaddingTop);
-  }
-}
-
-// Search
-.#{$menu}__search {
-  padding-block-start: var(--#{$menu}__search--PaddingTop);
-  padding-block-end: var(--#{$menu}__search--PaddingBottom);
-  padding-inline-start: var(--#{$menu}__search--PaddingLeft);
-  padding-inline-end: var(--#{$menu}__search--PaddingRight);
+// - Menu list
+.#{$menu}__list :where(.#{$divider}:is(li)) {
+  margin-block-start: var(--#{$menu}__list--divider--MarginTop);
+  margin-block-end: var(--#{$menu}__list--divider--MarginBottom);
 }
 
-// List
-.#{$menu}__list {
-  @include pf-v5-hidden-visible(var(--#{$menu}__list--Display));
-
-  padding-block-start: var(--#{$menu}__list--PaddingTop);
-  padding-block-end: var(--#{$menu}__list--PaddingBottom);
-
-  > .#{$divider} {
-    margin-block-start: var(--#{$menu}__list--c-divider--MarginTop);
-    margin-block-end: var(--#{$menu}__list--c-divider--MarginBottom);
-  }
-}
-
-// List item
+// - Menu list item
 .#{$menu}__list-item {
-  @include pf-v5-hidden-visible(var(--#{$menu}__list-item--Display));
-
-  position: relative;
+  align-items: start;
+  min-width: 0; // allow list item to shrink for text overflow
   color: var(--#{$menu}__list-item--Color);
   background-color: var(--#{$menu}__list-item--BackgroundColor);
```

Comment:
Are either of these needed?

Reply:
`background-color` is needed, `color` is not